.changeset/brave-garlics-lead.md

```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-documentation': minor
+'@swisspost/design-system-components': minor
```

Comment:
You simply updated the docs but did not add a feature or page, therefore it is a simple patch. 
On the contrary, this change to the header will require projects to update there code once its published so it's a breaking change for the components package. 
```suggestion
'@swisspost/design-system-documentation': patch
'@swisspost/design-system-components': major
```